pkg/handler/eventhandler_test.go

```diff
@@ -819,6 +840,28 @@ var _ = Describe("Eventhandler", func() {
 			Expect(actualRequests).To(Equal([]reconcile.Request{{NamespacedName: types.NamespacedName{Name: "my-pod"}}}))
 		})
 
+		It("should not lower the priority of a create request for an object that was created less than one minute in the past without the WithLowPriority wrapperr", func() {
```

Comment:
nit:
wrapperr => wrapperr

Please also do one search replace here for WithLowPriority => WithLowPriorityWhenUnchanged

(would have ignored this one, but as there's also another finding.. :))

Reply:
Maybe let's ignore this comment, because it should be resolved after => https://github.com/kubernetes-sigs/controller-runtime/pull/3152#discussion_r1992789734